setup.py

```diff
@@ -18,7 +30,7 @@
       author='Kevin Paul',
       author_email='kpaul@ucar.edu',
       url='https://github.com/NCAR/ASAPPyTools',
-      license='https://github.com/NCAR/ASAPPyTools/blob/master/LICENSE.rst',
+      license='https://github.com/NCAR/ASAPPyTools/blob/master/LICENSE.txt',
```

Comment:
I realize its confusing (like the rest of Python packaging) but the recommended way of identifying your license so both automated tools and humans can easily read it is instead of using the `license` field (which is intended only for custom/bespoke licenses, though I don't think there's explicit harm in including a link here), add the appropriate [trove classifier](https://pypi.org/pypi?%3Aaction=list_classifiers) (in this case, `License :: OSI Approved :: Apache Software License`) to the `classifiers` argument to `setup()`, e.g. `classifers=['License :: OSI Approved :: Apache Software License'],`. You might also find to add a few other classifiers too to help people find your project more easily, like those describing the Python versions you support (e.g. `Programming Language :: Python :: 2.7`, `Programming Language :: Python :: 3`), the general function of the package (e.g. `Topic :: Utilities` and `Topic :: Scientific/Engineering :: Atmospheric Science`), etc.

Reply:
Yeah.  That's another thing that should be added.  Good point!